plugins/processors/ifname/ifname_test.go

```diff
@@ -0,0 +1,106 @@
+//+build localsnmp
+
+package ifname
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/internal"
+	"github.com/influxdata/telegraf/internal/snmp"
+	"github.com/influxdata/telegraf/testutil"
+	"github.com/stretchr/testify/require"
+)
+
+func _TestTable(t *testing.T) {
```

Comment:
Why the leading underscore?

Reply:
The underscore is there to disable the test temporarily.  I didn't mean to leave it in.